### PR TITLE
chore(*): regenerate release notes for June 21 2021 release

### PR DESCRIPTION
**Pull request by pavelgj**

### CODE REVIEW

**Review by jhuleatt — APPROVED** [review]
@pavelgj can you please add a note about Cloud Functions runtime to each changelog? To make it a little easier, you can use the main tracking issue (#660) instead of each individual extension's update PR:

```
feature - updated Cloud Functions runtime to Node.js 14 (#660)
```

Besides that, LGTM!

File: firestore-bigquery-export/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+## Version 0.1.14
+
+feature - added warsaw (europe-central2) location (#677)
+
+fixed - Updated file naming for bq imports (#589)
```
Comment (jhuleatt) [inline firestore-bigquery-export/CHANGELOG.md:5]: The file naming PR was only a change to the npm script for firestore-bigquery-export, not the extension itself, so please remove this line. More context [here](https://github.com/firebase/extensions/pull/650#issuecomment-828536860)

File: delete-user-data/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,9 @@
+## Version 0.1.9
+
+feature - added warsaw (europe-central2) location (#677)
```
Comment (jhuleatt) [inline delete-user-data/CHANGELOG.md:3]: nit - consider capitalizing Warsaw in each changelog

File: firestore-bigquery-export/CHANGELOG.md (hunk 1 above)
Comment (pavelgj) [reply firestore-bigquery-export/CHANGELOG.md]: Done.

File: delete-user-data/CHANGELOG.md (hunk 2 above)
Comment (pavelgj) [reply delete-user-data/CHANGELOG.md]: Done.

**Pull request merged**